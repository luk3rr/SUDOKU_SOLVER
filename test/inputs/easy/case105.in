074100000 001009005 050030010 100003000 030520070 000970100 000400096 400207000 920006400 
