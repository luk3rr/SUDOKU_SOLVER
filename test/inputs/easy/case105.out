Solving the following grid:
0 7 4 | 1 0 0 | 0 0 0 
0 0 1 | 0 0 9 | 0 0 5 
0 5 0 | 0 3 0 | 0 1 0 
------+-------+------
1 0 0 | 0 0 3 | 0 0 0 
0 3 0 | 5 2 0 | 0 7 0 
0 0 0 | 9 7 0 | 1 0 0 
------+-------+------
0 0 0 | 4 0 0 | 0 9 6 
4 0 0 | 2 0 7 | 0 0 0 
9 2 0 | 0 0 6 | 4 0 0 


Solution found :')

3 7 4 | 1 5 2 | 6 8 9 
2 8 1 | 7 6 9 | 3 4 5 
6 5 9 | 8 3 4 | 7 1 2 
------+-------+------
1 9 7 | 6 4 3 | 5 2 8 
8 3 6 | 5 2 1 | 9 7 4 
5 4 2 | 9 7 8 | 1 6 3 
------+-------+------
7 1 3 | 4 8 5 | 2 9 6 
4 6 5 | 2 9 7 | 8 3 1 
9 2 8 | 3 1 6 | 4 5 7 

Algorithm: BFS
Total time: 66 ms
Total expanded states: 9387
